lib/Doctrine/DBAL/Event/Listeners/SqliteSessionInit.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
```

Comment:
Please remove license header, we're going to drop it everywhere (already done in develop in #2853).